### PR TITLE
DEV: Rename settings to post voting

### PR DESCRIPTION
**Pull request by nattsw**
Renaming the site settings from qa_ to post_voting, since this plugin used to be called question-and-answer

Screenshot 2023-08-21 at 3 56 44 PM

### CODE REVIEW

**Review by ZogStriP — APPROVED** [review]

**Review by tyb-talks — APPROVED** [review]

File: config/locales/server.zh_CN.yml
Comment (CvX) [inline config/locales/server.zh_CN.yml:1]: I was checking if this change needs a `@discourse-translator-bot keep_translations_and_approvals` annotation in the commit message but I think it doesn't? (it doesn't change the existing translations - it removes them and adds new ones so no need to "keep" anything)

**Review by CvX — APPROVED** [review]

File: config/locales/server.zh_CN.yml
Comment (nattsw) [reply config/locales/server.zh_CN.yml:1]: Cool, I didn't know we had this feature with translator bot

Comment (gschlager) [reply config/locales/server.zh_CN.yml:1]: Hmm, nope. @CvX is correct. Renaming translation keys will remove/add the translations on Crowdin, so no instructions for the bot are needed. Crowdin will use translation memory to pre-translate the new translations if only the key and not the string changes, so we should be good. There's nothing we can do about approvals. They will be gone, but that's nothing to worry about.

**Comment by nattsw** [comment]
😽 Thanks yall!

**Pull request merged**